webhook_features/src/features/summary_comment.rs

```diff
@@ -331,6 +331,7 @@ fn parse_review(comment: &str) -> Option<AckCommit> {
         .split('\n')
         .filter(|s| !s.starts_with('>')) // Ignore quoted text
         .flat_map(|s| s.split(|c: char| c.is_whitespace() || c.is_ascii_punctuation())) // Split on whitespace and punctuation
+        .flat_map(|s| s.split_whitespace())
```

Comment:
How is this different from `s.split(|c: char| c.is_whitespace())` above?

Reply:
I was confused as well: https://stackoverflow.com/questions/67559628/difference-between-split-and-split-whitespace

Reply:
> What about replacing this whole 100LOC function with a simple regex check?

I had trouble making it work with regexes originally but tried again in #10, let me know your thoughts :)